frontend/package.json

```diff
@@ -4,22 +4,35 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --config ./jest.config.json",
     "dev": "webpack --mode development ./src/index.js --output ./static/frontend/main.js",
     "build": "webpack --mode production ./src/index.js --output ./static/frontend/main.js",
     "watch": "watch 'npm run dev' ./src"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
+  "dependencies": {
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0"
+  },
   "devDependencies": {
     "@babel/core": "^7.8.4",
     "@babel/preset-env": "^7.8.4",
     "@babel/preset-react": "^7.8.3",
+    "axe-core": "^3.5.1",
     "babel-loader": "^8.0.6",
     "css-loader": "^3.4.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.18.3",
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
Whoa. I'd love to get an explanation of this sometime!

Reply:
@elliottyates agreed. As @dodget just explained to me, it's useful for handling CSS imports or other webpack imports when testing that would otherwise error out (see `jest.config.json`).

Looks like [identity-obj-proxy](https://github.com/keyz/identity-obj-proxy) is just a simple wrapper around  an  [ES6 Proxy Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy):

```
idObj = new Proxy({}, {
  get: function getter(target, key) {
    if (key === '__esModule') {
      return false;
    }
    return key;
  }
});
```

Example:

```
idObj.foo // -> "foo"
idObj.bar // -> "bar"
```